src/cljs/main/broadfcui/page/workspace/method_configs/method_config_editor.cljs

```diff
@@ -324,8 +360,8 @@
                         :headers utils/content-type=json
                         :on-done (fn [{:keys [success? get-parsed-response]}]
                                    (if success?
-                                     (swap! state assoc :loaded-config response :inputs-outputs (get-parsed-response))
-                                     (swap! state assoc :error (:message (get-parsed-response)))))}))
+                                     (swap! state assoc :loaded-config response :inputs-outputs (get-parsed-response) :redacted? false)
+                                     (swap! state assoc :loaded-config response :inputs-outputs (fake-inputs-outputs response) :redacted? true)))}))
```

Comment:
why not just pass `(:methodConfiguration response)` here rather than having to do it in `fake-inputs-outputs`?